schemas/uaprotocol_auto_add.py

```diff
@@ -0,0 +1,33 @@
+def extensionobject_from_binary(data):
```

Comment:
... and then created these two functions (they go to the end of uaprotocol_auto.py)...